bin/install-site.sh

```diff
@@ -160,8 +166,8 @@ deb http://the.earth.li/debian/ ${DISTVERSION} main contrib non-free
 deb-src http://the.earth.li/debian/ ${DISTVERSION} main contrib non-free
 
 # Security Updates:
-deb http://security.debian.org/ ${DISTVERSION}/updates main contrib non-free
-deb-src http://security.debian.org/ ${DISTVERSION}/updates main contrib non-free
+deb http://security.debian.org/ ${SECURITY} main contrib non-free
+deb-src http://security.debian.org/ ${SECURITY} main contrib non-free
```

Comment:
Either appears to work.

@sagepe should we add this path as it matches https://wiki.debian.org/DebianBullseye#FAQ It seems to be compatible Stretch and Buster too?

```suggestion
deb http://security.debian.org/debian-security ${SECURITY} main contrib non-free
deb-src http://security.debian.org/debian-security ${SECURITY} main contrib non-free
```

Reply:
Yes, why not, let's do that.

Reply:
Done